app/components/modals/org-file-cleanup-list/template.hbs

```diff
@@ -0,0 +1,37 @@
+<div local-class="file-cleanup-list-container">
+  <div local-class="list-table">
+    <div local-class="list-head">
+      <span>{{t 'date'}}</span>
+      <span>{{t 'type'}}</span>
+      <span>{{t 'fileCleanup.status'}}</span>
+    </div>
+    <div local-class="list-body">
+      {{#if this.isLoading}}
+        <div local-class="loading-rows">
+          <ThreeBounceSpinner></ThreeBounceSpinner>
+        </div>
+      {{else}}
+        {{#each this.fileCleanupList as |cleanup|}}
+          <div local-class="list-body__row">
+            <span local-class="list-body__row-item">{{moment-format cleanup.createdOn 'DD MMM YYYY'}}</span>
```

Comment:
Is there a ember way to do this? Moment is sort of deprecated and we are planning to move away from it.